LocalyticsXamarin/LocalyticsXamarin.Android/Additions/Localytics.cs

```diff
@@ -162,5 +163,59 @@ internal static bool __IsEmpty(IMessagingListenerV2Implementor value)
             }
         }
 
+        public static Func<LocalyticsXamarin.Android.Campaign, bool> ShouldPromptForLocationPermission;
```

Comment:
should fix it so JNI generation correctly generates this. May be later release.

Reply:
Mind explaining how that works?

Reply:
The code you have here could be  I assume is a copy paste modification or a generated modified and added here. All changes in Additions/Localytics.cs are supposed to be additional that cannot be generated by modifying the transform metadataxml. Ideally the generated code should be private or protected or internal and this interface exposes a public version that maps to the internal representation generated from the metadata.xml. 

I realize I had done a few of these because of naming conflicts between iOS and android. The intention was to fix this for a subsequent release.